src/SWP/SuperdeskBridgeBundle/README.md

```diff
@@ -0,0 +1,14 @@
+Superdesk Content API Bridge bundle
+==========
+
+Configuration options
+
+```yml
+    superdeskbridge:
```

Comment:
it should be swp_superdesk_bridge as is declared here: https://github.com/superdesk/web-renderer/pull/15/files#diff-e368ccdbac45264fc8197fda012fac3fR33